tests/test_bookmarks.py

```diff
@@ -341,7 +343,12 @@ def test_run(self):
                 expected_records = expected_records_2.get(stream)
                 primary_keys = stream_primary_keys.get(stream)
                 pk = list(primary_keys)[0] if primary_keys else None
-                if stream != 'orders':  # ORDERS has too many dependencies to track explicitly
+                if stream in {'orders', 'modifier_lists'}:  # Some streams have too many dependencies to track explicitly
```

Comment:
Later we should clean these lists of streams instead of maybe hardcoding them like this. Like maybe have constant sets at the top that describe what each one is / why. I bet if we look across different tests there are some patterns, but I could be wrong.

Reply:
Agreed, I think this can be done in a later PR. Will add it to the refactor/cleanup ticket in Jira.